lib/puppet/provider/sensu_ad_auth/sensuctl.rb

```diff
@@ -108,7 +104,7 @@ def flush
         if @property_flush[:server_binding]
           server['binding'] = @property_flush[:server_binding][host]
         else
-          server['binding'] = resource[:server_binding][host]
+          server['binding'] = resource[:server_binding][host] if resource[:server_binding]
```

Comment:
does this conditional have a test?

Reply:
Indirectly, yes.  I added more explicit test.